election-server/resources/schema.sql

```diff
@@ -16,5 +16,17 @@ CREATE TABLE IF NOT EXISTS ballots (
   verified BOOL NOT NULL
 );
 
+-- Table containing voting power delegations
+CREATE TABLE IF NOT EXISTS delegations (
+  id INT8 PRIMARY KEY, -- For pagination
+  transaction_hash BYTEA NOT NULL,
+  block_time TIMESTAMP WITH TIME ZONE NOT NULL,
+  from_account BYTEA NOT NULL UNIQUE,
```

Comment:
Do we overwrite existing delegations and thus delete any old ones from a given account?

Reply:
Yes, I don't see any reason to keep them around.. My reasoning is, that it will just make us more vulnerable to some guy registering loads of delegations from one account.